Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1179,6 +1179,22 @@
   categories:
     - UplinkUtility
 
+- type: listing
+  id: UplinkSyndicateToxinKit
+  name: uplink-syndicate-toxin-kit-name
+  description: uplink-syndicate-toxin-kit-desc
+  productEntity: ToxinKit
+  cost:
+    Telecrystal: 6
+  categories:
+  - UplinkJob
+  conditions:
+  - !type:BuyerJobCondition
+    whitelist:
+    - Bartender
+    - Chef
+    - MedicalDoctor
```

Comment:
why doesn't the chemist and the paramedic have that?

Reply:
should just be available to everyone yeah